packages/clay-multi-select/src/ClayMultiSelect.js

```diff
@@ -330,6 +337,15 @@ class ClayMultiSelect extends ClayComponent {
  * @type {!Object}
  */
 ClayMultiSelect.STATE = {
+	/**
+	 * Flag to indicate whether input can create item.
+	 * @default false
+	 * @instance
+	 * @memberof ClayMultiSelect
+	 * @type {?bool}
+	 */
+	allowInputCreateItem: Config.bool().value(true),
```

Comment:
Naming suggestion: should this be of the form `allow${NOUN}`? Something like `allowItemCreation`? (I'm not familiar with our naming conventions yet, but `allowInputCreateItem` struck me as a bit awkward.)

Reply:
Yes this may seem a bit confusing, but I just want to point out that input can create items. But we can create items from autocomplete or select, for example... Maybe that name was not so clear. I think that just going with `allowItemCreation` might seem like we will disable the creation of items in any user interaction. Makes sense 🤔?

Reply:
Not sure if too verbose, but maybe `allowItemCreationFromInput`?

Reply:
I do not see problem in going with that name despite being large but it is clearer and API is self explanatory.